### PR TITLE
Njs 4 ".env file"

### PR DESCRIPTION
**Pull request by DawidZlobecki**

### CODE REVIEW

File: app/index.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
+import {} from "dotenv/config";
```
Comment (JustD1d1t) [inline app/index.js:1]: import what?

**Review by JustD1d1t — APPROVED** [review]
LGTM

File: .env.example [hunk 2]
```diff
@@ -0,0 +1,3 @@
+PORT=8000
+DB_URL=LINK TO MONGODB
```
Comment (ruljin) [inline .env.example:2]: If in the example you add any values, it should be example values, defaults or no values. In this case example or no values wouls be good

File: app/index.js [hunk 3]
```diff
@@ -1,3 +1,5 @@
+import dotenv from "dotenv";
+dotenv.config();
```
Comment (ruljin) [inline app/index.js:2]: Never, leave usage lines between imports. Move it beneath them

**Review by ruljin — CHANGES_REQUESTED** [review]
Changes requested

**Comment by ruljin** [comment]
I can't comment `README.md` so I will do it here. Add table of contents and if you don't have any idea how to write this section, try basing on the `README.md` from the previous projects for example. The content should be more self-explenatory - what is `PORT` env used for? What secret for etc.

**Comment by ruljin** [comment]
My proposition.

Section name: `Environment variables`
Content of the section: ".env file is a source of environment variables for this application"
"PORT - blah, blah, blah,"
"DB_URL - blah, blah, blah,"
"JWT_SECRET - blah, blah, blah".

That's the last thing to change IMO.

**Pull request merged**